roles/docker_storage_verify/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# vim: set ft=ansible:
+#
+- name: Include distribution vars
+  include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_distribution|lower }}-{{ ansible_distribution_major_version }}.yml"
+    - "{{ ansible_distribution|lower }}.yml"
+
+- name: Fail if required variables are not defined
+  fail:
+    msg: "The required variables are not defined.  Check files under roles/docker_storage_verify/vars"
+  when: driver is undefined
+
+  # Maximum efficiency!  The output from 'docker info' contains a line:
+  #
+  #   Storage Driver: <value>
+  #
+  # So we grab the value, split on the colon + space, and compare the value
+  # to the expected driver.  All in one task!
+- name: Check Docker storage driver
+  shell: docker info | grep '^Storage Driver'
+  register: storage_driver
+  failed_when: "'{{ storage_driver.stdout.split(': ')[1] }}' != '{{ driver }}'"
```

Comment:
Man, I really wanted to use that but the `--format` flag on `docker info` is only available on 1.13

Once all the streams are on 1.13, I can go back and update this.

I figured a cleaner usage of `grep`, so I'm still going to push an update with that.